### ISSUE TITLE
The .exe seems to struggle with the API section

### ISSUE THREAD

**Issue by DandelionSprout** [phyziyx/steam2pcgw#1]
Testing with the app IDs 2124570 and 2178170, there seems to be some problems.

<details><summary>Command line log:</summary>

```
PS C:\Users\imree\Downloads\steam2pcgw-main\steam2pcgw-main> .\steam2pcgw.exe
Running Steam 2 PCGW Converter v0.0.62 ( https://github.com/phyziyx/steam2pcgw )
Insert the Steam app ID: 2124570
Sanitised text: '2124570' (len: 7)
Fetching game app details...
Cached the game...
* [1/24] Adding stub
* [2/24] Adding app cover
* [3/24] Adding app developers
* [4/24] Adding app publishers
* [5/24] Adding app release date
* [6/24] Adding reception score
* [6/24] Skipped Metacritic
* [7/24] Game is F2P
* [8/24] Taxonomy...
* [9/24] Processing introduction...
* [10/24] Processing Availability!
* [11/24] Processing Monetization!
* [12/24] Processing Microtransactions!
* [13/24] Processing DLCs!
* [14/24] Processing Config File Location!
* [15/24] Processing Save Game Location!
* [16/24] Processing Save Game Sync!
* [17/24] Processing Video!
* [18/24] Processing Input!
* [19/24] Processing Audio!
* [20/24] Processing Languages!
* [21/24] Processing API!
panic: runtime error: index out of range [1] with length 0

goroutine 1 [running]:
main.GetExeBit(0x698901, 0x6dbfb1, 0x7, 0xc000000101, 0xc000180ff0, 0x0, 0x0)
        D:/Programming/steam2pcgw/steam2pcgw/utils.go:45 +0x2cf
main.main()
        D:/Programming/steam2pcgw/steam2pcgw/main.go:477 +0x3a45
```
</details>

<details><summary>The .txt file that was generated for 2124570</summary>

```
{{stub}}

{{Infobox game
|cover        = GAME TITLE cover.jpg
|developers   = 
{{Infobox game/row/developer|EPIC LEAGUE PTE. LTD.}}
|publishers   = 
{{Infobox game/row/publisher|EPIC LEAGUE PTE. LTD.}}
|engines      = 
|release dates= 
{{Infobox game/row/date|Windows|14 Oct, 2022}}
|reception    = 
{{Infobox game/row/reception|Metacritic|link|rating}}
{{Infobox game/row/reception|OpenCritic|link|rating}}
{{Infobox game/row/reception|IGDB|link|rating}}|taxonomy     =
{{Infobox game/row/taxonomy/monetization      | Free-to-play }}
{{Infobox game/row/taxonomy/microtransactions | }}
{{Infobox game/row/taxonomy/modes             | Singleplayer }}
{{Infobox game/row/taxonomy/pacing            | }}
{{Infobox game/row/taxonomy/perspectives      | }}
{{Infobox game/row/taxonomy/controls          | }}
{{Infobox game/row/taxonomy/genres            | Action, Free to Play, RPG}}
{{Infobox game/row/taxonomy/sports            | }}
{{Infobox game/row/taxonomy/vehicles          | }}
{{Infobox game/row/taxonomy/art styles        | }}
{{Infobox game/row/taxonomy/themes            | }}
{{Infobox game/row/taxonomy/series            | PCGW Templates<!-- CHANGE TO THE ACTUAL SERIES NAME IF ONE EXISTS --> }}
|steam appid  = 2124570
|steam appid side = 
|gogcom id    = 
|gogcom id side = 
|official site= https://darkthrone.epicleague.io
|hltb         = 
|igdb         = <!-- Only needs to be set if there is no IGDB reception row -->
|lutris       = 
|mobygames    = 
|strategywiki = 
|wikipedia    = 
|winehq       = 
|license      = commercial
}}

{{Introduction
|introduction      = <strong>▶Dark Throne◀</strong><br>The heroes of the human world begin a great adventure to bring peace back to Izendar, the human land occupied by the Demon Realm.<br>Enjoy a variety of gameplay options, such as ever-growing demons, epic items, three different game modes, etc.<br><br>■ Authentic 'hack and slash' action role-playing game with<br>Battles with intense actions with thrill <br>Exhilarating pleasure of sweeping enemies away at once with exquisite skill action!<br><br>■ Various missions and hidden traps with:<br>Captivating battles in the demon world imbued with intricate plots and missions to complet<br>New dungeons being changed with infinite combinations<br>Boss stages requiring unique strategies<br><br>■ Charming heroes with various characteristics <br>Paladin, Assassin, Demon Hunter and more<br>A variety of weapons and armors, and action-skills<br><br>■ AAA quality game<br>Jeehyung Lee as Art Director, who is also the cover artist of renowned comic books<br>Dark Throne OST featuring K-Pop top producer Keeproots<br>Sound effects that maximize the immersion of the game<br><br>■ In-depth gameplay experience with<br>'Stress-free' game control with simple and easy operation<br>Meticulous UX designs

|release history      = Released on 14 Oct, 2022

|current state     = 
}}

==Availability==
{{Availability|{{Availability/row| Steam | 2124570 | Steam |  |  | Windows, MAC }}
}}
<!-- PAGE GENERATED BY PCGW2STEAM  -->


==Monetization==
{{Monetization
|ad-supported           = 
|dlc                    = 
|expansion pack         = 
|freeware               = 
|free-to-play           = true
|one-time game purchase = 
|sponsored              = 
|subscription           = 
}}

===Microtransactions===
{{Microtransactions
|boost               = 
|cosmetic            = 
|currency            = 
|finite spend        = 
|infinite spend      = 
|free-to-grind       = 
|loot box            = 
|none                = None
|player trading      = 
|time-limited        = 
|unlock              = 
}}

{{DLC|
<!-- DLC rows goes below: -->
}}

==Game data==
===Configuration file(s) location===
{{Game data|
{{Game data/config|Windows|}}
}}

===Save game data location===
{{Game data|
{{Game data/saves|Windows|}}
}}

===[[Glossary:Save game cloud syncing|Save game cloud syncing]]===
{{Save game cloud syncing
|discord                   = 
|discord notes             = 
|epic games launcher       = 
|epic games launcher notes = 
|gog galaxy                = 
|gog galaxy notes          = 
|origin                    = 
|origin notes              = 
|steam cloud               = 
|steam cloud notes         = 
|ubisoft connect           = 
|ubisoft connect notes     = 
|xbox cloud                = 
|xbox cloud notes          = 
}}

==Video=={{Video
|wsgf link                  = 
|widescreen wsgf award      = 
|multimonitor wsgf award    = 
|ultrawidescreen wsgf award = 
|4k ultra hd wsgf award     = 
|widescreen resolution      = unknown
|widescreen resolution notes= 
|multimonitor               = unknown
|multimonitor notes         = 
|ultrawidescreen            = unknown
|ultrawidescreen notes      = 
|4k ultra hd                = unknown
|4k ultra hd notes          = 
|fov                        = unknown
|fov notes                  = 
|windowed                   = unknown
|windowed notes             = 
|borderless windowed        = unknown
|borderless windowed notes  = 
|anisotropic                = unknown
|anisotropic notes          = 
|antialiasing               = unknown
|antialiasing notes         = 
|upscaling                  = unknown
|upscaling tech             = 
|upscaling notes            = 
|vsync                      = unknown
|vsync notes                = 
|60 fps                     = unknown
|60 fps notes               = 
|120 fps                    = unknown
|120 fps notes              = 
|hdr                        = unknown
|hdr notes                  = 
|ray tracing                = unknown
|ray tracing notes          = 
|color blind                = unknown
|color blind notes          = 
}}

==Input=={{Input

|key remap                 = unknown
|key remap notes           = 
|acceleration option       = unknown
|acceleration option notes = 
|mouse sensitivity         = unknown
|mouse sensitivity notes   = 
|mouse menu                = unknown
|mouse menu notes          = 
|invert mouse y-axis       = unknown
|invert mouse y-axis notes = 
|touchscreen               = unknown
|touchscreen notes         = 
|controller support        = false
|controller support notes  = 
|full controller           = 
|full controller notes     = 
|controller remap          = unknown
|controller remap notes    = 
|controller sensitivity    = unknown
|controller sensitivity notes= 
|invert controller y-axis  = unknown
|invert controller y-axis notes= 
|xinput controllers        = unknown
|xinput controllers notes  = 
|xbox prompts              = unknown
|xbox prompts notes        = 
|impulse triggers          = unknown
|impulse triggers notes    = 
|dualshock 4               = unknown
|dualshock 4 notes         = 
|dualshock prompts         = unknown
|dualshock prompts notes   = 
|light bar support         = unknown
|light bar support notes   = 
|dualshock 4 modes         = unknown
|dualshock 4 modes notes   = 
|tracked motion controllers= unknown
|tracked motion controllers notes = 
|tracked motion prompts    = unknown
|tracked motion prompts notes = 
|other controllers         = unknown
|other controllers notes   = 
|other button prompts      = unknown
|other button prompts notes= 
|controller hotplug        = unknown
|controller hotplug notes  = 
|haptic feedback           = unknown
|haptic feedback notes     = 
|simultaneous input        = unknown
|simultaneous input notes  = 
|steam input api           = unknown
|steam input api notes     = 
|steam hook input          = unknown
|steam hook input notes    = 
|steam input presets       = unknown
|steam input presets notes = 
|steam controller prompts  = unknown
|steam controller prompts notes = 
|steam cursor detection    = unknown
|steam cursor detection notes = 
}}==Audio==
{{Audio
|separate volume           = unknown
|separate volume notes     = 
|surround sound            = unknown
|surround sound notes      = 
|subtitles                 = unknown
|subtitles notes           = 
|closed captions           = unknown
|closed captions notes     = 
|mute on focus lost        = unknown
|mute on focus lost notes  = 
|royalty free audio        = false
|eax support               = 
|eax support notes         = 
|red book cd audio         = 
|red book cd audio notes   = 
|general midi audio        = 
|general midi audio notes  = 
}}{{L10n|content={{L10n/switch
|language  = Korean
|interface = true
|audio     = false
|subtitles = true
|notes     = 
|fan       = 
|ref       = }}{{L10n/switch
|language  = English
|interface = true
|audio     = true
|subtitles = true
|notes     = 
|fan       = 
|ref       = }}{{L10n/switch
|language  = Japanese
|interface = true
|audio     = false
|subtitles = true
|notes     = 
|fan       = 
|ref       = }}{{L10n/switch
|language  = Simplified Chinese
|interface = true
|audio     = false
|subtitles = true
|notes     = 
|fan       = 
|ref       = }}}}\n\n

==Other information==
===API===
{{API
```
</details>

Tested with PowerShell 7.2.7 on Windows 11 22H2 x64.

There's other problems too, including lack of newlines where there should be some, "OS X" being rendered as "MAC" in all-caps, and that it'd probably be better to remove the "PCGW Templates" series name by default.

**Comment by phyziyx** [comment]
Hi, thanks for creating the issue!

I had fixed the error that you reported in an earlier build, the other errors mentioned have been resolved.  Can you highlight where specifically the 'OS X' is being rendered as 'MAC' so I can verify PCGW template and the Steam API?

Thanks again.

**Comment by DandelionSprout** [comment]
>Can you highlight where specifically the 'OS X' is being rendered as 'MAC' so I can verify PCGW template and the Steam API?

It seemed to occur in this line as of 4 hours ago: `{{Availability|{{Availability/row| Steam | 2124570 | Steam |  |  | Windows, MAC }}`

It also appears to occur in this section as of 0.0.66:
```
{{System requirements
|OSfamily = Mac
```

**Comment by phyziyx** [comment]
Resolved in #2.
